src/System.Net.NetworkInformation/src/System/Net/StartIPOptions.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net.NetworkInformation
+{
+    [Flags]
+    internal enum StartIPOptions { Both = 3, None = 0, StartIPv4 = 1, StartIPv6 = 2 }
```

Comment:
To clarify: The file used in the Windows project is actually under the Common directory. This file was accidentally kept here when I was working initially off of the snn branch, just to get things compiling. I deleted this extra copy in a subsequent commit but I'm not sure if GitHub is going to remove it from here.